### DISCUSSION TITLE
Improvement to NPC Indicators (menu highlight colors)

### DISCUSSION THREAD

**Discussion by lineyyy** [runelite/runelite#16769] [Ideas]
Currently, there is a toggle for highlighting menu names, which changes the menu color to whatever you have set for the highlight color in the NPC indicators plug in. 
Earlier this year or last year, there was a RL update that made it so NPC highlight color could be individually changed (which is fantastic!). However, the menu highlight color for tagged NPCs still defaults to the highlight color in the NPC indicators plug in (does not change if you select a different color for a specific NPC). 
Would it be possible to add a feature that allows the menu highlight color to reflect the color you’ve set an NPC to instead of defaulting to the single color selected within the plug in?

**Comment by Nightfirecat** [comment]
Fixed in 034f79029cd7dc0f942b7af17e0e530b9abeafc1; you'll see this working as expected in our next release.
